csunplugged/topics/models.py

```diff
@@ -18,7 +18,7 @@ def __str__(self):
         """Text representation of GlossaryTerm object.
 
         Returns:
-            Term attribute of GlossaryTerm (string).
+            Term attribute of GlossaryTerm (str).
```

Comment:
Since we were using `dict` for dictionary, it seemed appropriate to use `str` for string. However specifying the type is only required in the description by the style guide, what would you like to do?

- Full text of type in brackets?
- Python notation of type in brackets?
- Mention type in description?
- Something else?